packages/augur-ui/src/modules/create-market/landing.tsx

```diff
@@ -82,7 +82,7 @@ export default class Landing extends React.Component<LandingProps> {
                     tertiary: '',
                   },
                   categoryStats
-                )}
+                ).sort((a, b) => (a.count > b.count ? -1 : 1))}
```

Comment:
I don't tgibk we want the cards to hump around based on number of markets created in the cat. I’d rather them stay in alpha order.